### PR TITLE
Updates the New visualization info

### PR DESCRIPTION
**Pull request by JV0812**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request...

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/search/get-started-with-search/search-basics/chart-search-results.md [hunk 1]
```diff
@@ -25,17 +25,13 @@ For this example, you can see a bar chart, but you can pick from any of the avai
...
-
-<img src={useBaseUrl('img/search/get-started-search/search-page/new-visualizations.png')} alt="new-visualizations" style={{border:'1px solid black'}} width="800" />
+New Visualizations will enable the new chart visualizations for your log search, also helping you to customize each chart by modifying the available settings and providing a unified experience across the Sumo Logic platforms.
 
 **Data** represents the data that has been used to plot chart, **Round data point** toggle button is used to turn ON/OFF the round off to the value in tooltip, and the **Visual Settings** options allow you to customize your chart. The other customizing properties include **[Display Overrides](/docs/dashboards/panels/modify-chart/#overridedashboard-displays)**, **[Axes](/docs/dashboards/panels/modify-chart/#modifychart-axes)**, **[Legend](/docs/dashboards/panels/modify-chart/#modifychart-legend)**, **Thresholds**, and **[JSON](/docs/dashboards/panels/modify-chart/#modifychart-legend)**.
 
 <img src={useBaseUrl('img/search/get-started-search/search-basics/log-query-charts-properties.png')} alt="log-query-charts-properties" style={{border:'1px solid black'}} width="800" />
```
Comment (kimsauce) [inline docs/search/get-started-with-search/search-basics/chart-search-results.md:32]:
```suggestion
The **New Visualization** feature enables the new chart visualizations for your log search, allowing you customize each chart by modifying the available settings and providing a unified experience across the Sumo Logic platforms.

* **Data**. This section represents the data that has been used to plot the chart. Toggle the **Round data point** ON/OFF the round off to the value in tooltip,.
* **Visual Settings**. These options allow you to further customize your chart: **Color Palette**, **Fill Opacity**, **Marker Type**, and **Dimensions** (x, y, and z).

<! -- insert New Visualizations screenshot -->
```

File: docs/search/get-started-with-search/search-basics/chart-search-results.md [hunk 2]
```diff
@@ -25,17 +25,14 @@ For this example, you can see a bar chart, but you can pick from any of the avai
...
+* **Data**. This section represents the data that has been used to plot the chart. Toggle the **Round data point** ON/OFF the round off to the value in tooltip,.
+* **Visual Settings**. These options allow you to further customize your chart: **Color Palette**, **Fill Opacity**, **Marker Type**, and **Dimensions** (x, y, and z).
 
 <img src={useBaseUrl('img/search/get-started-search/search-basics/log-query-charts-properties.png')} alt="log-query-charts-properties" style={{border:'1px solid black'}} width="800" />
 
-If you prefer to use the classic visualizations UI, you can do so by clicking the **Classic Visualizations** button. You'll be asked to confirm your choice, Click **Switch to Classic Visualizations** to proceed.
+Charts will appear in new visualization style by default. If you prefer to use the classic visualizations UI, you can do so by clicking the **Classic Visualizations** button. You'll be asked to confirm your choice, Click **Switch to Classic Visualizations** to proceed.  However, if you choose a classic visualization, your selection will be remembered, and any further searches in the same or new tab will always render with the classic visualization.
```
Comment (kimsauce) [inline docs/search/get-started-with-search/search-basics/chart-search-results.md:35]:
```suggestion
Charts will appear in **New Visualization** style by default. If you prefer to use the **Classic Visualization** UI, click the **Classic Visualizations** button, then in the popup window, click **Switch to Classic Visualizations** to proceed.  

:::tip
If you choose **Classic Visualization**, your selection will be remembered, and any further searches in the same or new tab will always render with the classic visualization.
:::
```

File: docs/search/get-started-with-search/search-basics/chart-search-results.md [hunk 3]
```diff
@@ -25,17 +25,18 @@ For this example, you can see a bar chart, but you can pick from any of the avai
...
-
-<img src={useBaseUrl('img/search/get-started-search/search-page/new-visualizations.png')} alt="new-visualizations" style={{border:'1px solid black'}} width="800" />
-
-**Data** represents the data that has been used to plot chart, **Round data point** toggle button is used to turn ON/OFF the round off to the value in tooltip, and the **Visual Settings** options allow you to customize your chart. The other customizing properties include **[Display Overrides](/docs/dashboards/panels/modify-chart/#overridedashboard-displays)**, **[Axes](/docs/dashboards/panels/modify-chart/#modifychart-axes)**, **[Legend](/docs/dashboards/panels/modify-chart/#modifychart-legend)**, **Thresholds**, and **[JSON](/docs/dashboards/panels/modify-chart/#modifychart-legend)**.
+* **Data**. This section represents the data that has been used to plot the chart. Toggle the **Round data point** ON/OFF the round off to the value in tooltip,.
+* **Visual Settings**. These options allow you to further customize your chart: **Color Palette**, **Fill Opacity**, **Marker Type**, and **Dimensions** (x, y, and z).
```
Comment (kimsauce) [inline docs/search/get-started-with-search/search-basics/chart-search-results.md:31]:
```suggestion
* **Visual Settings**. These options allow you to further customize your chart: [**Color Palette**](/docs/dashboards/panels/modify-chart/#modifydisplay-settings), [**Fill Opacity**](/docs/dashboards/panels/modify-chart/#modifydisplay-settings), [**Marker Type**](/docs/dashboards/panels/modify-chart/#modifydisplay-settings), and [**Dimensions**](/docs/dashboards/panels/modify-chart/#modifychart-axes) (x, y, and z).
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
Please ignore my comment about the screenshot - yours is all good! Sorry for the confusion. I just had one last comment...

**Comment by JV0812** [comment]
its okay Kim. Thanks for your review.

**Review by kimsauce — APPROVED** [review]

File: docs/search/get-started-with-search/search-basics/chart-search-results.md [hunk 4]
```diff
@@ -25,17 +25,18 @@ For this example, you can see a bar chart, but you can pick from any of the avai
 
 ### New Visualizations
```
Comment (kimsauce) [inline docs/search/get-started-with-search/search-basics/chart-search-results.md:26]:
```suggestion
### New Visualization
```

File: docs/search/get-started-with-search/search-basics/chart-search-results.md [hunk 5]
```diff
@@ -23,19 +23,20 @@ Follow the below instructions to chart aggregate the results:
...
-
-**Data** represents the data that has been used to plot chart, **Round data point** toggle button is used to turn ON/OFF the round off to the value in tooltip, and the **Visual Settings** options allow you to customize your chart. The other customizing properties include **[Display Overrides](/docs/dashboards/panels/modify-chart/#overridedashboard-displays)**, **[Axes](/docs/dashboards/panels/modify-chart/#modifychart-axes)**, **[Legend](/docs/dashboards/panels/modify-chart/#modifychart-legend)**, **Thresholds**, and **[JSON](/docs/dashboards/panels/modify-chart/#modifychart-legend)**.
+* **Data**. This section represents the data that has been used to plot the chart. Toggle the **Round data point** ON/OFF the round off to the value in tooltip,.
+* **Visual Settings**. These options allow you to further customize your chart: [**Color Palette**](/docs/dashboards/panels/modify-chart/#modifydisplay-settings), [**Fill Opacity**](/docs/dashboards/panels/modify-chart/#modifydisplay-settings), [**Marker Type**](/docs/dashboards/panels/modify-chart/#modifydisplay-settings), and [**Dimensions**](/docs/dashboards/panels/modify-chart/#modifychart-axes) (x, y, and z).
```
Comment (JV0812) [inline docs/search/get-started-with-search/search-basics/chart-search-results.md:31]:
```suggestion
* **Visual Settings**. Allows you to further customize your chart. The other customizing properties include: [**Color Palette**](/docs/dashboards/panels/modify-chart/#modifydisplay-settings), [**Fill Opacity**](/docs/dashboards/panels/modify-chart/#modifydisplay-settings), [**Marker Type**](/docs/dashboards/panels/modify-chart/#modifydisplay-settings), and [**Dimensions**](/docs/dashboards/panels/modify-chart/#modifychart-axes) (x, y, and z).
```

**Pull request merged**